sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsMeta.scala

```diff
@@ -1098,6 +1098,7 @@ abstract class BaseExprMeta[INPUT <: Expression](
   //+------------------------+-------------------+-----------------------------------------+
   lazy val needTimezoneCheck: Boolean = {
     wrapped match {
+      case _: FromUnixTime => false
```

Comment:
why do we still need ```FromUnixTime``` given it's a sub-class of ```TimeZoneAwareExpression```?

Reply:
Done, now use new APIs.